docs/requirements.md

```diff
@@ -1,16 +1,59 @@
-# PerfCheck CI Requirements
+# Perf Check CI Requirements
 
 Perf Check CI is a web application to manage jobs that measure and compare performance on another web application.
 
 ## Creating a job
 
-* As a user I want to create a job to compare the performance of two branches across multiple request paths.
+* As a user, I want to compare relative performance of my branch against master to make sure I'm not shipping code that will slow things down.
+* As a user, I want to have a quick idea how performance on my branch looks, without caring about comparisons against master.
+* As a user, I want to create a job to compare the performance of two branches across multiple request paths.
+* As a user, I know the request I'm working on takes a long time (10-60 seconds), so just want a very rough comparison that takes a smaller amount of time to complete so I can get back to work.
```

Comment:
What is the pertinent different between this and `quick idea how performance on my branch looks`? Would this requirement exist if benchmarking would only take 5 seconds?

Reply:
The “quick idea” is not comparative. It’s test branch only and used for iterating, usually when you already know the values for master/reference. 

The “long request” version is comparative but for example instead of running 20 requests on both test/reference, it’s 5-10. 

It’s essentially a way to specify ‘n’ should be adaptive without specifying that a user needs to directly specify the actual quantity.

Reply:
I think I "see" your main point. I'm updating language, but keeping the two separate as they are different use cases and different ways to cut time down. If everything took 5 seconds, we wouldn't have to build this tool at all :D Feel free to push back though.